TheWarWithin/Priorities/RogueAssassination.simc

```diff
@@ -7,7 +7,7 @@ actions.precombat+=/variable,name=trinket_sync_slot,value=2,if=trinket.2.has_sta
 actions.precombat+=/variable,name=effective_spend_cp,value=cp_max_spend-2<?5*talent.hand_of_fate
 # Pre-cast Slice and Dice if possible
 actions.precombat+=/stealth
-actions.precombat+=/slice_and_dice,precombat_seconds=1,if=refreshable&!buff.indiscriminate_carnage.up
+## actions.precombat+=/slice_and_dice,precombat_seconds=1,if=refreshable&!buff.indiscriminate_carnage.up
```

Comment:
The default sim priority (assuming a fresh, clean boss fight) still manually uses Slice and Dice before combat.  The modifications above were made to prevent it from refreshing prematurely every time you get out of combat.  My only real concern here is that `deathmark_condition` checks that Slice and Dice will be up for 5+ seconds, so omitting this could impact the opener / delay Deathmark potentially.

(Not a required change, just making sure we've thought it through.)

Reply:
> The default sim priority (assuming a fresh, clean boss fight) still manually uses Slice and Dice before combat. The modifications above were made to prevent it from refreshing prematurely every time you get out of combat. My only real concern here is that `deathmark_condition` checks that Slice and Dice will be up for 5+ seconds, so omitting this could impact the opener / delay Deathmark potentially.
> 
> (Not a required change, just making sure we've thought it through.)

I asked this question directly to the APL writer for Assassination and he said delete it

Reply:
![image](https://github.com/user-attachments/assets/ea04ab1b-1b69-43e3-b18b-982c93c695ae)